packages/engine/Source/Scene/GaussianSplat3DTileContent.js

```diff
@@ -80,6 +81,43 @@ function GaussianSplat3DTileContent(loader, tileset, tile, resource) {
   this._transformed = false;
 }
 
+/**
+ * Performs checks to ensure that the provided tileset has the Gaussian Splatting extensions.
+ *
+ * @param tileset The tileset to check for the extensions.
+ * @returns {boolean} Returns <code>true</code> if the necessary extensions are included in the tileset.
+ * @static
+ */
+GaussianSplat3DTileContent.tilesetHasGaussianSplattingExt = function (tileset) {
+  let hasGaussianSplatExtension = false;
+  let hasLegacyGaussianSplatExtension = false;
+  if (tileset.isGltfExtensionRequired instanceof Function) {
+    hasGaussianSplatExtension =
+      tileset.isGltfExtensionRequired("KHR_gaussian_splatting") &&
+      tileset.isGltfExtensionRequired(
+        "KHR_gaussian_splatting_compression_spz_2",
+      );
+
+    // TODO: Remove support for deprecated experimental extension for CesiumJS 1.135 release.
+    //  See: https://github.com/CesiumGS/cesium/issues/12842
```

Comment:
Please remove these comments. [We track work in GitHub issues, not in `TODO` comments in the code](https://github.com/CesiumGS/cesium/tree/main/Documentation/Contributors/CodingGuide#:~:text=TODO%20comments%20need%20to%20be%20removed%20or%20addressed%20before%20the%20code%20is%20merged%20into%20main.%20Used%20sparingly%2C%20PERFORMANCE_IDEA%2C%20can%20be%20handy%20later%20when%20profiling.).